plugin/src/main/java/org/xtclang/plugin/XtcProjectDelegate.java

```diff
@@ -159,6 +159,9 @@ public Void apply(final Void args) {
 
         // This is all config phase. Warn if a project isn't versioned when the XTC plugin is applied, so that we
         // are sure no skew/version conflicts exist for inter-module dependencies and cross publication.
+        if ("test".equals(projectName)) {
```

Comment:
I don't think you meant this to be committed :)

Reply:
Nope sorry. Fixing